src/views/Hearing/Sidebar.js

```diff
@@ -35,6 +37,27 @@ class Sidebar extends React.Component {
     );
   }
 
+  getLanguageChanger() {
+    const {hearing, activeLanguage, dispatch} = this.props;
+    const availableLanguages = { fi: 'Kuuleminen Suomeksi', sv: 'Frågorna tillgängliga', en: 'Questionaire in English'};
```

Comment:
Fix the text to `{ fi: 'Kuuleminen suomeksi', sv: 'Enkäten på svenska', en: 'Questionnaire in English'}`